terragrunt/Makefile

```diff
@@ -0,0 +1,8 @@
+.PHONY: fmt checkov
+
+fmt:
```

Comment:
If you are going to have a make file, I would probably add plan and validate into the mix.